R/ranges.R

```diff
@@ -196,6 +196,37 @@ singleGenome <- function(x) {
 }
 
 
+seqlengthsFromAPI <- function(genome) {
```

Comment:
As we discussed, all of these helper functions can probably go away.

Reply:
Yes, I'll update and push the changes.